src/game/Laptop/Laptop.cc

```diff
@@ -3263,10 +3265,9 @@ static void HandleWebBookMarkNotifyTimer(void)
 void ClearOutTempLaptopFiles(void)
 {
 	// clear out all temp files from laptop
-	FileDelete("files.dat");
-	FileDelete("finances.dat");
-	FileDelete("email.dat");
```

Comment:
just for anyone else reading this: the file is never created.

Reply:
Yes. I have looked through the code for quite some time, because I thought the path might be created by building a string or something, but I have not seen any indications of that, or the file ever being created...